frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -47,6 +51,22 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
     this.layers = layers;
   }
 
+  getMinScale(): number {
+    return this.minScale;
+  }
+
+  setMinScale(min: number): void {
+    this.minScale = min;
+  }
```

Comment:
Should we ensure that min < max?
Although this kind of validation may be problematic when using two separate values vs an array extent such as `[number, number]` because when trying to set new values, you'll have to pay attention to the order in which they are set otherwise an error can be thrown ;/

Still unsure if i prefer an array extent over individual values.

Reply:
Hmmm... guess it would be better to make it an array.